lib/instr.mli

```diff
@@ -23,7 +23,7 @@ module type S = sig
   val tr : InstrLit.t -> t
   val nop : t option
   val is_nop : t -> bool
-  val is_overwritable : t -> bool
+  val needs_cmodx : t -> bool
```

Comment:
Maybe this deserve a comment to explain what it is about? Even if to be fair this whole file would deserve a bit of documentation

Reply:
Thanks for pointint this out. I added a comment there and tried to change the function name to hopefully look less of a cryptic slang.